src/PhpSpec/Process/Shutdown/ShutdownActionInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Process\Shutdown;
+
+interface ShutdownActionInterface
+{
+    public function runAction();
+
+    public function output();
```

Comment:
Not all shutdown actions will output something, this does not need to be part of the interface